pkgs/by-name/fo/fop/package.nix

```diff
@@ -1,61 +1,43 @@
 {
   lib,
-  stdenv,
   fetchurl,
-  fetchpatch,
-  ant,
-  jdk,
+  maven,
   jre,
   makeWrapper,
   stripJavaArchivesHook,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+maven.buildMavenPackage rec {
   pname = "fop";
-  version = "2.8";
+  version = "2.11";
 
   src = fetchurl {
-    url = "mirror://apache/xmlgraphics/fop/fop-${finalAttrs.version}-src.tar.gz";
-    hash = "sha256-b7Av17wu6Ar/npKOiwYqzlvBFSIuXTpqTacM1sxtBvc=";
+    url = "https://dlcdn.apache.org/xmlgraphics/fop/source/fop-${version}-src.tar.gz";
+    hash = "sha256-uY6cUjmyuenfK3jAWvugsYa5qg8rbnvRZZ6qA/g2fZM=";
   };
 
-  patches = [
-    (fetchpatch {
-      name = "CVE-2024-28168.patch";
```

Comment:
This patch has been part of Apache FOP since 2.10 https://github.com/apache/xmlgraphics-fop/commit/d96ba9a11710d02716b6f4f6107ebfa9ccec7134

Reply:
Put this into the commit message? (And wrap commit message body at 72 chars.)